btc/path_multisig_wallet.go

```diff
@@ -144,18 +141,62 @@ func (b *backend) pathMultiSigWalletRead(ctx context.Context, req *logical.Reque
 }
 
 func (b *backend) GetMultiSigWallet(ctx context.Context, store logical.Storage, walletName string) (*multiSigWallet, error) {
-	entry, err := store.Get(ctx, PathWallet+walletName)
+	entry, err := store.Get(ctx, PathMultiSigWallet+walletName)
 	if err != nil {
 		return nil, err
 	}
 	if entry == nil {
 		return nil, nil
 	}
 
-	var w multiSigWallet
-	if err := entry.DecodeJSON(&w); err != nil {
+	var raw map[string]interface{}
+	if err := entry.DecodeJSON(&raw); err != nil {
 		return nil, err
 	}
 
-	return &w, nil
+	return toMultiSig(raw)
+}
+
+func toMultiSig(raw map[string]interface{}) (*multiSigWallet, error) {
```

Comment:
it is called from getWalletFromType in `path_transaction.go`.
```golang
type multisigWallet struct {
   *wallet,
   M int,
   N int,
   PubKeys []string
}
```
Since now `multisigWallet` extends `wallet` type, we can't use the JSON response because it cointains the "squashed" multisigWallet type, i.e:
```
{
   "M",
   "N",
   "PubKeys",
   "Mnemonic",         //
   "DerivationPath",   //  -> wallet
   "Network",          //
}
```